### ISSUE TITLE
no such file or directory when converting from tslint to eslint

### ISSUE THREAD

**Issue by captainmorgan0** [angular-eslint/angular-eslint#555]
I'm running the command
ng g @angular-eslint/schematics:convert-tslint-to-eslint --remove-tslint-if-no-more-tslint-targets --ignore-existing-tslint-config MYPROJECTNAME

And I'm getting this error 
ENOENT: no such file or directory, mkdir 'C:\Users\ANDREW~1.DA\AppData\Local\Temp\tmp-21888-pX3HoNzl8zW1'

My username fold has a is Andrew.D'Ambrosio
Could any of those characters in it cause a problem? Can I specify an alternative location?

**Comment by captainmorgan0** [comment]
Turns out the issue is indeed the ' in my folder name.
I got around it by changing my TEMP path

**Comment by mkurte** [comment]
Hey guys,

I'm getting the same error, but I'm pretty sure this is not an issue with my current working directory.

I type `ng g @angular-eslint/schematics:convert-tslint-to-eslint PROJECTNAME`, check the defaults and get this error:
```
There was a critical error when trying to inspect your tslint.json:

env: node: No such file or directory
```

I'm running this command on a mac, in a multiple projects workspace.
My project is in the root folder, the `tslint.json` file too. Is there something I am missing?

Many thanks in advance,
Matthias

**Comment by JamesHenry** [comment]
@captainmorgan0 Glad you found a resolution, I'm not sure if there is anything we can do in this toolkit to support that directly, so I'll close this for now.

@mkurte This does not seem to be related to this issue, so please open a new one if you are still having trouble. It makes it hard to follow and reason about if unrelated things are put into the same thread as one another.

It's very unusual but that error suggests you have a malformed `tslint.json` because that error comes when TSLint itself gives us that feedback.

You should be able to reproduce using TSLint directly by running:

```sh
npx tslint --print-config {PATH_TO_YOUR_TSLINT_JSON_HERE}
```

Hopefully that helps you to track down what you have in there that TSLint doesn't like. As I say though, feel free to open a new issue with as much information as possible if you get stuck
